AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Organization/Edit.cshtml

```diff
@@ -57,17 +57,17 @@
         <h3>Organization Information</h3>
 
         <div class="form-group">
-            <label asp-for="LogoUrl" class="col-md-2 control-label"></label>
+            <label asp-for="Name" class="col-md-2 control-label">Organisation Name</label>
```

Comment:
On this the label text will actually get rendered on the backend based on the properties of our model class. PR #2252 is a good example of this. Are you able to adjust in this case?

Reply:
No problem